### PR TITLE
Add new workshops

### PR DESCRIPTION
**Pull request by jianoway**

### CODE REVIEW

File: src/data/workshops.js [hunk 1]
```diff
@@ -213,11 +213,119 @@ const INDUSTRY_WORKSHOPS = [
     moderators: [],
     image: '/workshops/nft-state-of-art.jpg',
   },
+  {
+    id: 12,
+    date: '19 Jan 2022',
+    time: '2000-2100',
+    title: 'Crypto for Decentralized\nSystems',
```
Comment (jerrenss) [inline src/data/workshops.js:220]: Is the '\n' there on purpose? I think don't need it right

File: src/data/workshops.js [hunk 2]
```diff
@@ -213,11 +213,119 @@ const INDUSTRY_WORKSHOPS = [
...
+      },
+    ],
+    moderators: [],
+    image: '/workshops/crystal-center-3.jpg',
+  },
+  {
+    id: 15,
+    date: '20 Jan 2022',
+    time: '2000-2100',
+    title: 'Algorithmic Scalability of Blockchain protocols',
```
Comment (jerrenss) [inline src/data/workshops.js:274]: Algorithmic Scalability of Blockchain Protocols

File: src/data/workshops.js [hunk 3]
```diff
@@ -213,11 +213,119 @@ const INDUSTRY_WORKSHOPS = [
...
+      },
+    ],
+    moderators: [],
+    image: '/workshops/crystal-center-5.jpg',
+  },
+  {
+    id: 17,
+    date: '21 Jan 2022',
+    time: '2100-2200',
+    title: 'Security of Layer-0 blockchain networks',
```
Comment (jerrenss) [inline src/data/workshops.js:310]: Security of Layer-0 Blockchain Networks

**Review by jerrenss — COMMENTED** [review]
Looks good! Just minor naming changes required, thanks!

File: src/data/workshops.js (hunk 1 above)
Comment (jianoway) [reply src/data/workshops.js:220]: "Decentralized Systems" was appearing on a single line, causing the word "Systems" to be slightly cropped out in the card so I added a line break to paragraph the words. Is this okay?

Comment (jerrenss) [reply src/data/workshops.js]: Oh isit? Hmm I tried removing the line break locally and it looks fine without haha. Try remove first? See if the preview got the crop issue

Comment (jerrenss) [reply src/data/workshops.js]: Cuz I think the text wrap is automatic

**Comment by jerrenss** [comment]
Thanks!

**Review by jerrenss — APPROVED** [review]

**Pull request merged**